easybuild/easyconfigs/c/cairo/cairo-1.14.4-intel-2015b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'cairo'
+version = '1.14.4'
+
+homepage = 'http://cairographics.org'
+description = """Cairo is a 2D graphics library with support for multiple output devices.
+ Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
+ PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB"""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['http://cairographics.org/releases/']
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libpng', '1.6.18'),
```

Comment:
I started out doing that, but then I ran into dep hell... several of the other dependencies here also include `libpng`, including ones that we already have, so I would need to make `-libpng-1.6.19` versions for them, and then make sure those are used everywhere